docs/admin/project.md

```diff
@@ -0,0 +1,84 @@
+  <figcaption>Project page</figcaption>
+</figure>
+
+This page will list all the projects in a cluster, their name, owner and when its quota was last updated. By clicking on the _edit configuration_ link of a project you will be able to edit the quotas of that project.
+
+<figure>
+  <img src="../../assets/images/admin/projects/project_quotas.png" alt="Project quotas" />
+  <figcaption>Project quotas</figcaption>
+</figure>
+
+### Storage
+
+Storage quota represents the amount of data a project can store. The storage quota is broken down in three different areas:
+
+- **Feature Store**: This represents the storage quota for files and directories stored in the `_featurestore.db` dataset in the project. This dataset contains all the feature group offline data for the project.
+- **Hive DB**: This represents the storage quota for files and directories stored in the `[projectName].db` dataset in the project. This is a general purpose Hive database for the project that can be used for analytics.
+- **Project**: This represents the storage quota for all the data stored on any ohter dataset.
+
+Each storage quota is divided into space quota, i.e. how much space the files can consume, and namespace quota, i.e. how many files and directories there can be. If Hopsworks is deployed on-premise using hard drives to store the data, i.e. Hopsworks is not configured to laverage a S3-compliant storage system, the data is replicated across multiple nodes (by default 3) and the space quota takes the replication into consideration. As an example, a 100MB file stored with a replication factor of 3, will consume 300MB of space quota.
+
+By default, all storage quotas are disabled and not enforced. Administrators can change this default by changing the following configuration in the [Configuration](../admin/variables.md) UI and/or the cluster definition:
+```
+hopsworks:
+    featurestore_default_quota: [default quota in bytes, -1 to disable]
+    hdfs_default_quota: [default quota in bytes, -1 to disable]
+    hive_default_quota: [default quota in bytes, -1 to disable]
+```
+The values specified will be set during project creation and administrators will be able to customize each project using this UI.
+
+### Compute
+
+Compute quotas represents the amount of compute a project can use to run Spark and Flink applications as well as Tez queries. Quota is expressed as number of seconds a container of size 1 CPU and 1GB of RAM can run for.
+
+If the Hopsworks deployment is connected to a Kubernetes cluster, the Python jobs, Jupyter notebooks and KServe models are not subject to the compute quota. Currently Hopsworks does not support defining quotas for compute happening on the connected Kubernetes cluster.
+
+By default, the compute quota is disabled and each project can execute as many jobs as it needs. Administrators can change this default by changing the following configuration in the [Condiguration](../admin/variables.md) UI and/or the cluster definition:
```

Comment:
By default, the compute quota is disabled and each project can execute as many jobs as it needs.
->
By default, the compute quota is disabled.

(the other text doesn't add info)